experiments/generic/disk_fill/disk_fill_ansible_logic.yml

```diff
@@ -0,0 +1,74 @@
+---
+- hosts: localhost
+  connection: local
+  vars_files: 
+    - /chaoslib/litmus/disk_fill/test_vars_diskfill.yml
+  vars:
+    c_util: "/chaoslib/litmus/disk_fill/disk_fill_by_litmus.yml"
+    c_container: "{{ lookup('env','TARGET_CONTAINER') }}"
+    c_experiment: "disk-fill"
+    c_duration: "{{ lookup('env','TOTAL_CHAOS_DURATION') }}"
+    a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+    a_label: "{{ lookup('env','APP_LABEL') }}"
+    fill_percentage: "{{ lookup('env','FILL_PERCENTAGE') }}"
+    
+  tasks:
+    - block:
+
+        ## GENERATE EXP RESULT NAME
+        - block:
+
+            - name: Construct chaos result name (experiment_name)
+              set_fact:
+                c_experiment: "{{ lookup('env','CHAOSENGINE') }}-{{ c_experiment }}"
+
+          when: lookup('env','CHAOSENGINE')    
+
+        ## RECORD START-OF-EXPERIMENT IN LITMUSCHAOS RESULT CR
+        - include_tasks: /utils/runtime/update_chaos_result_resource.yml
+          vars:
+            status: 'SOT'
+            namespace: "{{ a_ns }}"
+
+        ## PRE-CHAOS APPLICATION LIVENESS CHECK
+
+        - name: Verify that the AUT (Application Under Test) is running 
+          include_tasks: /utils/common/status_app_pod.yml
+          vars:
+            app_ns: "{{ a_ns }}" 
+            app_lkey: "{{ a_label.split('=')[0] }}"
```

Comment:
Can you directly pass label without splitting it into key and value?

Reply:
@shubhamchaudhary `utils/common/status_app_pod.yml` is using the label in split form i.e app_lkey and app_lvalue that's why i guess it is needed to be splitted.

Reply:
@uditgaurav , we can change the utils/common/status_app_pod.yml file accordingly as i already mentioned https://github.com/litmuschaos/litmus/pull/851#discussion_r340298488